lib/kdump_utils.pm

```diff
@@ -112,6 +112,16 @@ sub activate_kdump {
     } until (match_has_tag('yast2_console-finished'));
 }
 
+sub activate_kdump_without_yast {
+    # activate kdump by grub, need a reboot to start kdump
+    my $cmd = "if [ -e /etc/default/grub ]; then sed -i '/^GRUB_CMDLINE_LINUX_DEFAULT/ s/\"\$/ crashkernel=256M,high crashkernel=128M,low \"/' /etc/default/grub; fi";
+    script_run($cmd);
+    script_run("cat /etc/default/grub");
+    # sync changes from /etc/default/grub into /boot/grub2/grub.cfg
+    assert_script_run("grub2-mkconfig -o /boot/grub2/grub.cfg");
+    assert_script_run("systemctl enable kdump.service");
```

Comment:
```suggestion
    systemctl('enable kdump.service');
```
there is special function for this

Reply:
@asmorodskyi Thanks, taken. "Commit suggestion" is a good function in github, but I don't know how to merge commits into one. So I edit it and force push a new version :)

Reply:
this actually how I expecting this happen , not sure that fancy github features will suit our flow ...